crates/holochain/src/conductor/conductor.rs

```diff
@@ -2075,7 +2080,7 @@ mod scheduler_impls {
             let cell_arcs = {
                 let mut cell_arcs = vec![];
                 for cell_id in self.live_cell_ids() {
-                    if let Ok(cell_arc) = self.cell_by_id(&cell_id).await {
+                    if let Ok(cell_arc) = self.cell_by_id(&cell_id, true).await {
```

Comment:
I don't know if this should be false, but could imagine so.

Reply:
That makes sense actually, updated